pkgs/ifupdown-ng.nix

```
@@ -0,0 +1,35 @@
+{
+  fetchFromGitHub,
+  stdenv,
+  libbsd,
+  iproute2,
+  ...
+}:
+stdenv.mkDerivation {
+  pname = "ifupdown-ng";
+  version = "0.12.1";
+
+  src = fetchFromGitHub {
+    owner = "ifupdown-ng";
+    repo = "ifupdown-ng";
+    rev = "e305296b3d23e56bba92c504e030d8e4b91db403";
+    sha256 = "1zf04waaklqvq4ywxyjswwpxpmq7jrbjkw3yd8aapxfj20gkph56";
```

```suggestion
    sha256 = "1zf04waaklqvq4ywxyjswwpxpmq7jrbjkw3yd8aapxfj20gkph56";
```

This should use `hash`, not `sha256`.